README.md

```diff
@@ -127,15 +127,15 @@ handle_event(<<"decr">>, #{}, #{assigns := Assigns} = Socket) ->
 %% --------------------------------------------------------------------
 
 counter(Macros) ->
-    ?ARIZONA_LIVEVIEW(~s"""
+    ?ARIZONA_LIVEVIEW(~"""
```

Comment:
I was not aware that `~s"""` returns an `iolist()` and `~"""` a UTF-8 binary. For me, both would return a UTF-8 binary. This change is in README.md, which needs a complete revisit. Anyway, I'm changing here to remember this since the scanner expects a binary to be parsed.

Reply:
I wouldn't worry too much about the README now. I'm sure some other stuff's broken already. And the less changes we make in pull requests the easier they are to review 😄